articles/quickstart/spa/_includes/_callback_component.md

```diff
@@ -1,3 +1,6 @@
-You can choose to have users return to any URL in your application that you like; however, it is recommended that you create a dedicated callback route to serve as a central location that the user will be returned to upon successful authentication. Having a single callback route is beneficial for two main reasons:
-* It prevents the need to whitelist multiple (and sometimes unknown) callback URLs
-* It serves as a place to show a loading indicator while your application sets the user's client-side session
+::: panel Select a callback URL
+You can select any URL in your application for your users to return to. We recommend creating a dedicated callback route. 
```

Comment:
The style guide tells us to avoid using both passive tense and 'we', but I felt like using 'we recommend' is better than 'is recommended'. Is there a preference?